docs/events/format.md

```diff
@@ -0,0 +1,39 @@
+# <event_name>
+
+## Intro
+<event_name> - one sentence description of the event
+
+## Description
+Detailed description of the event. Should include:
+* What is the purpose of the event?
+* Are there any edge-cases, drawbacks or advantages of using it?
+
+## Arguments
+* `<arg#1>`[<tags>] - short description of the argument value.
```

Comment:
Add argument type

Reply:
Should elaborate on possible changes to the argument's value and type (like with arg-parser)